op-node/rollup/driver/driver.go

```diff
@@ -264,5 +269,16 @@ func NewDriver(
 		altSync:       altSync,
 	}
 
+	// gather and wire up actors
```

Comment:
Losing bunch of isolation. Each actors may procedurally invoked by other actors.